python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -57,16 +58,19 @@ def evaluate(
         group_metrics_min_queries: int = 50,
         logs_dir: Optional[str] = None,
         logging_frequency: int = 25,
+        track_experiment: bool = False,
     ):
         """
         Evaluate the ranking model
 
         Args:
             test_dataset: an instance of tf.data.dataset
             inference_signature: If using a SavedModel for prediction, specify the inference signature
+            additional_features: Additional post processing feature functions as key value pairs
+            group_metrics_min_queries: Minimum number of queries per group to be used for group aggregate metrics
+            logs_dir: Directory to log the predictions and metrics
             logging_frequency: integer representing how often(in batches) to log status
-            metric_group_keys: list of fields to compute group based metrics on
-            save_to_file: set to True to save predictions to file like self.predict()
+            track_experiment: Boolean to determine if weights and biases tracking should be used
```

Comment:
@lastmansleeping Should we also take the mentions of `metric_group_keys` and `save_to_file` out of `base/model/relevance_model.py` ? I don't see them implemented anywhere [anymore?].

Reply:
Yep. I will do that.